database/migrations/2024_09_16_115228_create_personal_access_tokens_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
migration file automatically added when I installed Sanctum, feel free to skip review on this